### DISCUSSION TITLE
Is it possible to open the tab opened from search bar (not address bar) as a sibling tab?

### DISCUSSION THREAD

**Discussion by zeroarst** [piroor/treestyletab#3225] [Q&A] [answered]
I know that address bar one cannot be implemented as already dicusssed before. But I would like to how about the search bar?
Currently it opens as an independent tab at the bottom. 
Is it possible to add a key combination such as alt + enter to open the tab as a sibling tab?
![image](https://user-images.githubusercontent.com/1834837/194677995-7a976c09-d717-4823-a2e1-66b4d9c39a82.png)

**Comment by irvinm** [comment, marked as answer]
For the address bar, you can create sibling tabs by setting:

`user_pref("browser.tabs.insertAfterCurrent", true);`

I would imagine this should work for the search bar, but I have not confirmed.

**Reply by piroor** [reply]
Additional information: TST has no option to do that due to Firefox's restriction - Firefox doesn't provide ability for addons to know an opened tab is from the search bar or not. Thus you need to use a generic way like this.
